utils/cleanup_mocked_prometheus_response.py

```diff
@@ -0,0 +1,89 @@
+response = requests.get(url)
+if response.status_code == 200:
+    print("Successfully downloaded!")
+    lines = [line.decode() for line in response.content.splitlines()]
+    node = extract_property(lines, 'node')
+    apiserver_request_count = 0
+    result = [line for line in lines
```

Comment:
Some magic? :D Based on what this magic if happened? This would need to be edited if we poll new data, including the previous "url"?

Reply:
I didn't want to write mock data manually from scratch, it would miss a lot of properties etc. So instead I took data from existing environment and filter it out. And this script is filtering this out. It needs to be run everytime we start scraping new data, and I documented the process of updating this here https://github.com/solarwinds/swi-k8s-opentelemetry-collector/blob/b00b7159c52f54d63c108c7a72c080fb75910fd3/doc/development.md#updating-mocked-data-if-new-data-are-scraped 

But now I see I forgot to remove specific prometheushost that I used from code, I will fix that